tests/unit/utils/search-pluto-lots-test.js

```diff
@@ -5,6 +5,6 @@ module('Unit | Utility | search pluto lots');
 
 // Replace this with your real tests.
 test('it works', function(assert) {
-  let result = searchPlutoLots();
+  let result = searchPlutoLots('120 broadway');
```

Comment:
This works fine, but in #38 I simply set a default value for the utility (which is a blank string). Thinking out loud here, but this is where [TypeScript](https://en.wikipedia.org/wiki/TypeScript) comes in handy by giving our JS strongly typed classes.